examples/micronaut/build.gradle

```diff
@@ -39,7 +39,10 @@ dependencies {
 
 compileJava.options.compilerArgs += '-parameters'
 compileTestJava.options.compilerArgs += '-parameters'
-run.main = 'example.micronaut.Application'
+mainClassName = 'example.micronaut.Application'
 
 /** CHANGE THIS TO THE IMAGE YOU WANT TO PUSH TO */
 // jib.to.image = 'gcr.io/PROJECT_ID/micronaut-jib'
+
+// not needed once https://github.com/GoogleContainerTools/jib/issues/1501 is fixed
+jib.container.mainClass = mainClassName
```

Comment:
I believe we can remove this if we upgrade the version to `1.0.1`

Reply:
We still should probably use `mainClassName` directly as part of our main class inference though.

Reply:
Oh yeah we have to do the PR to update version numbers, I'll get on that

Reply:
OK, I can wait until then. I think we still need to change `run.main` to `mainClassName` to fix #1495.